projects/packages/protect-status/src/class-scan-status.php

```diff
@@ -0,0 +1,362 @@
+<?php
```

Comment:
This file was copied over from the Protect plugin. No changes other than the 'use'  statements for `\Protect_Models\...`.